client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-task/style.scss

```diff
@@ -1,4 +1,32 @@
 .confirmation-task__card {
+	--subtitle-font-size: 0.8125rem;
```

Comment:
It feels like we should try to push this into a variable much higher up, possibly with a Woo-related name. But I think we can tackle that in a follow-up if you prefer. Having this be defined so locally is a recipe for longer-term trouble.